pkg/node/node.go

```diff
@@ -166,7 +168,10 @@ func NewNode(
 			config.NodeID,
 			apiServer,
 			config,
-			bacalhauConfig.Metrics,
+			// TODO(review): we not longer have a config for this, and the default was always DevNull, can we remove this?
+			types.MetricsConfig{
+				EventTracerPath: os.DevNull,
+			},
```

Comment:
@wdbaruni can we remove this service given its never used (by default)?

Reply:
yeah makes sense